packages/cln_plugin/lib/src/json_rpc/response.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'error.dart';
+
+part 'response.g.dart';
+
+@JsonSerializable(explicitToJson: true)
+class Response {
+  /// A String specifying the version of the JSON-RPC protocol.
+  /// MUST be exactly "2.0".
+  String jsonrpc;
+
+  /// This structure contains additional information about the
+  /// result returned from the function.
+  dynamic result;
+
+  /// This field contains information about an unsuccessful call
+  /// and stores information about it.
+  Error error;
```

Comment:
```suggestion
  Error? error;
```

I like the class here instead of the `Map<String, dynamic>`, the only think here is to keep the Error nullable with `?`

Reply:
Done.